charts/jobset/templates/controller/deployment.yaml

```diff
@@ -62,18 +62,21 @@ spec:
         - name: webhook-cert
           mountPath: /tmp/k8s-webhook-server/serving-certs
           readOnly: true
+        {{- if .Values.certManager.enable }}
+        - mountPath: /tmp/k8s-metrics-server/metrics-certs
```

Comment:
Would this work if someone specifies a different location than this in CertDir?

Reply:
no it would not, they would have to modify the deployment

Reply:
So we should maybe mention this in our documentation.

Otherwise I worry this will be a support issue.

Reply:
This is the same approach we use with the webhook cert dir. Do we have to support such changes (i.e. custom deployments)? And we have documentation in the config API.